demo/fit_a_line/train_ft.py

```diff
@@ -0,0 +1,69 @@
+import paddle.v2 as paddle
+import os
+import gzip
+import glob
+import recordio
+import cPickle as pickle
+from paddle.v2.reader.creator import cloud_reader
+import paddle.v2.dataset.uci_housing as uci_housing
+
+master_ip = os.getenv("MASTER_IP")
```

Comment:
Maybe it's better to do:
```python
etcd_ip = os.getenv("ETCD_IP")
etcd_endpoint = "http://" + etcd_ip + ":" + "2379"

# ...

export MASTER_IP=$(python /root/k8s_tools.py fetch_master_ips)
export ETCD_IP="$MASTER_IP"
```
We can later just change the implementation of `export ETCD_IP="$MASTER_IP"`.